unified-runtime/source/adapters/native_cpu/common.cpp

```diff
@@ -11,21 +11,14 @@
 #include "common.hpp"
 
 // Global variables for UR_RESULT_ADAPTER_SPECIFIC_ERROR
-// See urGetLastResult
-thread_local ur_result_t ErrorMessageCode = UR_RESULT_SUCCESS;
+thread_local int32_t ErrorMessageCode;
```

Comment:
In other adapters you keep the initialization but change it to zero, here you drop the initialization, presumably it should also be kept here?